src/bp-forums/groups.php

```diff
@@ -1390,6 +1494,119 @@ public function replies_pagination( $args ) {
 			return $args;
 		}
 
+		/**
+		 * Ensure that forum content associated with a BuddyBoss group can only be viewed via the group URL.
+		 *
+		 * @since BuddyBoss 1.7.6
+		 *
+		 * @return void
+		 */
+		public function forum_redirect_canonical() {
```

Comment:
Move this function to the end of the file.

Reply:
Move the at the end of the file.
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/b2c663fb81f0630d02690c0ac5064d62ff9f8387